synapse/config/emailconfig.py

```diff
@@ -294,6 +294,9 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
             self.email_riot_base_url = email_config.get(
                 "client_base_url", email_config.get("riot_base_url", None)
             )
+            self.delay_before_mail_ms = Config.parse_duration(
+                email_config.get("delay_before_mail", "10m")
+            )
```

Comment:
I think keeping the comments from the constant is useful:

```suggestion
            # The amount of time we always wait before ever emailing about a notification
            # (to give the user a chance to respond to other push or notice the window)
            self.delay_before_mail_ms = Config.parse_duration(
                email_config.get("delay_before_mail", "10m")
            )
```